### ISSUE TITLE
Generator for angular components命令报错

### ISSUE THREAD

**Issue by mick-cs** [njleonzhang/generator-gulp-ionic#5]
`yo gulp-ionic:factory factory`

`module.js:471
    throw err;
    ^

Error: Cannot find module '../generator-base'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-gulp-ionic/generators/factory/index.js:4:23)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)`

**Comment by njleonzhang** [comment]
@mick-cs 谢谢，马上修复。稍等

**Comment by njleonzhang** [comment]
@mick-cs, 问题已经修复，刚release了version 1.2.1
请重新安装generator，应该就好了

```
npm install -g generator-gulp-ionic
```

**Comment by mick-cs** [comment]
在生成项目的时候会报这个错误
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/usr/local/lib/node_modules/generator-gulp-ionic/generators/app/templates/.gitignore'
    at Error (native)
    at Object.fs.openSync (fs.js:634:18)
    at Object.fs.readFileSync (fs.js:502:33)
    at bulkCopy (/usr/local/lib/node_modules/generator-gulp-ionic/node_modules/yeoman-generator/lib/actions/actions.js:65:17)
    at module.exports.yeoman.Base.extend._processDirectory (/usr/local/lib/node_modules/generator-gulp-ionic/generators/app/index.js:32:14)
    at module.exports.yeoman.Base.extend.writing (/usr/local/lib/node_modules/generator-gulp-ionic/generators/app/index.js:79:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-gulp-ionic/node_modules/yeoman-generator/lib/base.js:431:23)
    at /usr/local/lib/node_modules/generator-gulp-ionic/node_modules/run-async/index.js:26:25
    at /usr/local/lib/node_modules/generator-gulp-ionic/node_modules/run-async/index.js%

**Comment by njleonzhang** [comment]
TNX for your help. The issue is fixed in 1.2.5, please have a try.

```
npm install -g generator-gulp-ionic
```

**Comment by njleonzhang** [comment]
It should be OK, close.
